analyses/summary/types.go

```diff
@@ -0,0 +1,17 @@
+package summary
+
+// SeverityType is Possible Types of Severities from Server
+type SeverityType struct {
+	Low      int `json:"low"`
+	Medium   int `json:"medium"`
+	High     int `json:"high"`
+	Critical int `json:"critical"`
+}
+
+// StackSummary is SA Result Summary output
+type StackSummary struct {
```

Comment:
@prashbnair Need your input for the summary part.

Reply:
This looks fine as per the epic but can you take a sample output, add it to the epic and as Parag?